misc/scripts/install_d3d12_sdk_windows.py

```diff
@@ -1,11 +1,16 @@
 #!/usr/bin/env python
 
 import os
+import platform
 import shutil
+import ssl
 import subprocess
 import sys
 import urllib.request
 
+# Check if '-ssl_allow_unverified' is in the command line arguments
+if "-ssl_allow_unverified" in sys.argv:
+    ssl._create_default_https_context = ssl._create_unverified_context
```

Comment:
Use standard double hyphen syntax for command line arguments, and rename it to make explicit that an insecure option is used:

```suggestion
if "--insecure-ignore-ssl-verification" in sys.argv:
	# Ignore SSL certificate verification (insecure).
    ssl._create_default_https_context = ssl._create_unverified_context
```

(This is standard practice across many programs that support CLI arguments to ignore SSL certification, so it stands out more in CI files.)